sale_loyalty_auto_refresh/migrations/post-migration.py

```diff
@@ -0,0 +1,18 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+def update_parameter_key(env):
+    parameters = env["ir.config_parameter"].search(
+        [("key", "like", "sale_coupon_auto_refresh.")]
+    )
+    for parameter in parameters:
+        parameter.key = (parameter.key).replace(
```

Comment:
```suggestion
        parameter.key = parameter.key.replace(
```

Reply:
just curious but what's different or benefit of this suggested change?

Reply:
Better readability :smile: parenthesis are redundant in this case